.github/workflows/ansible-test.yml

```diff
@@ -24,7 +24,7 @@ jobs:
           python-version: 3.6
 
       - name: Install ansible-base (devel)
-        run: pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check
+        run: 'for i in {1..5}; do [ "${i}" -ne 1 ] && sleep 5; pip install https://github.com/ansible/ansible/archive/devel.tar.gz --disable-pip-version-check && break; done'
```

Comment:
Have you seen failures installing from `https://github.com/ansible/ansible/archive/devel.tar.gz`?